lib.nix

```diff
@@ -11,9 +11,12 @@ let
         if (value.type or "" == "derivation")
         then {
           "${typeField}" = "nixDerivation";
-          drvPath = value.drvPath;
-          outputName =
-            value.outputName;
+          inherit
+            (value)
+            drvPath
+            version
```

Comment:
Not all the packages have a `version` field, which breaks (in particular) the `c` template: https://github.com/nickel-lang/nickel-nix/actions/runs/5069730717/jobs/9103769608?pr=67